### PR TITLE
add openelb-joins-cncf-sandbox-project.md

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Signed-off-by: FeynmanZhou <feynman@kubesphere.io>

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
See [preview link](https://deploy-preview-2143--kubesphere-v3-1.netlify.app/zh/blogs/openelb-joins-cncf-sandbox-project/).

File: content/zh/blogs/openelb-joins-cncf-sandbox-project.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- 支持 VIP Group
+- 支持 IPv6
+- 提供独立的界面管理与配置 EIP 与 IP Pool
+- 集成至 KubeSphere Console；提供 Prometheus metrics 
+
+OpenELB 还将重点开展社区运营并推出一系列社区活动，希望借助更多开发者和用户的力量，解决用户在私有环境下的服务暴露与 IP 管理问题，为应用在 Kubernetes 上打开一扇大门，使服务对外暴露与管理变得更加轻松。
+
+## 持续开源开放
+
+KubeSphere 团队秉持 ”Upstream first“ 的原则，今年 7 月份先将 Fluentbit Operator 项目捐给 Fluent 社区成为 CNCF 子项目，此次又将 OpenELB 加入 CNCF Sandbox。未来 KubeSphere 团队将继续保持开源、开放的理念，持续作为 OpenELB 项目的参与方之一，推动国内和国际开源组织的生态建设，帮助 OpenELB 社区培育一个中立的开源社区与生态，与更多的容器平台及上下游生态伙伴进行深度合作，欢迎大家关注、使用 OpenELB 以及参与社区贡献。
```
Comment (Bettygogo2021) [inline content/zh/blogs/openelb-joins-cncf-sandbox-project.md:91]: The quotation is incorrect.
”Upstream first“ > “Upstream first”

**Review by Bettygogo2021 — CHANGES_REQUESTED** [review]

File: content/zh/blogs/openelb-joins-cncf-sandbox-project.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+---
+title: 'OpenELB 项目进入 CNCF Sandbox，让私有化环境对外暴露服务更简单'
+tag: 'CNCF'
+keyword: 'OpenELB, 负载均衡器, Kubernetes, LoadBalancer, 物理机'
+description: 'CNCF 宣布由青云科技 KubeSphere 团队开源的负载均衡器插件 OpenELB 正式进入 CNCF 沙箱'
```
[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply content/zh/blogs/openelb-joins-cncf-sandbox-project.md:5]: It is necessary when using Chinese.

File: content/zh/blogs/openelb-joins-cncf-sandbox-project.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+11 月 10 日，云原生计算基金会 (CNCF) 宣布由青云科技 KubeSphere 团队开源的负载均衡器插件 OpenELB 正式进入 CNCF 沙箱（Sandbox）托管。
+
+![示意图](https://kubesphere-community.pek3b.qingstor.com/images/8471636692467_.pic_hd.jpg)
+
+OpenELB 项目在此前命名为 PorterLB，是为物理机（Bare-metal）、边缘（Edge）和私有化环境设计的负载均衡器插件，可作为 Kubernetes、K3s、KubeSphere 的 LB 插件对集群外暴露 “LoadBalancer” 类型的服务，核心功能包括：
+
+- 基于 BGP 与 Layer 2 模式的负载均衡 
+- 基于路由器 ECMP 的负载均衡
+- IP 地址池管理管理
```
Comment (Bettygogo2021) [inline content/zh/blogs/openelb-joins-cncf-sandbox-project.md:21]: Delete "管理"

**Review by Bettygogo2021 — CHANGES_REQUESTED** [review]

**Comment by FeynmanZhou** [comment]
@Done. Please help to review and merge it. Thanks!

**Pull request merged**